db/service_updates.yml

```diff
@@ -1,3 +1,10 @@
+- date: '2022-05-09'
+  title: HESA records now visible in Register for April Census update
+  content: |
+    The import from HESA to Register is working normally. This means if you’re a Higher Education Institute (HEI), you can use Register to see data that you have updated and uploaded to HESA. This will allow you to complete your April Census update by 13 May.
+
+    If you identify any further issues with your data, contact [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
```

Comment:
Can we make the email subject line: April census update - HESA import issues in Register
(So when they click on the link, the email subject is automatically set) - However, not sure you can do that in the news and updates section - if not, don't worry!